### DISCUSSION TITLE
DLKZMK11LM Template modification

### DISCUSSION THREAD

**Discussion by z-x-f** [Koenkk/zigbee2mqtt#9481] [Devices]
I purchased an Aqara single switch module T1 (with neutral wire) (DLKZMK11LM), and my current configuration can also work normally and is more concise. Can I modify the existing template?
```js
    {
        zigbeeModel: ['lumi.switch.n0acn2'],
        model: 'DLKZMK11LM',
        vendor: 'Xiaomi',
        description: 'Aqara single switch module T1 (with neutral)',
        fromZigbee: [fz.on_off, fz.xiaomi_power, fz.xiaomi_switch_type, fz.xiaomi_switch_opple_basic],
        toZigbee: [tz.on_off, tz.xiaomi_power, tz.xiaomi_switch_type, tz.xiaomi_switch_power_outage_memory, tz.xiaomi_led_disabled_night],
        exposes: [e.switch(), e.power().withAccess(ea.STATE_GET), e.energy(), e.temperature().withAccess(ea.STATE),
            e.voltage().withAccess(ea.STATE), e.power_outage_memory(), e.led_disabled_night(), e.switch_type()],
        configure: async (device, coordinatorEndpoint, logger) => {
            await device.getEndpoint(1).write('aqaraOpple', {'mode': 1}, {manufacturerCode: 0x115f, disableResponse: true});
        },
    },
```
![dlkzmk11lm](https://user-images.githubusercontent.com/31797512/140095012-039812c4-0dd1-4719-9bb3-8f78944ee588.PNG)

**Comment by Koenkk** [comment]
If all features work like before this is fine, if yes please make a pr

**Comment by Alex3880** [comment]
Hello, did not find topic with Aqara DLKZMK11LM relay.
Have question why this device recognized as end device as it has neutral wire?
<img width="1013" alt="image" src="https://user-images.githubusercontent.com/12990053/160812700-ed1f5b03-0aa3-48f3-8936-7d6dfc1d07b7.png">

**Reply by Alex3880** [reply]
Using z2m version 1.24.0

**Reply by z-x-f** [reply]
Please use the latest usb stick firmware.  
I have encountered this problem before, and I repair it many times before it was successful.

**Reply by Alex3880** [reply]
Already updated. Using Sonoff zigbee 3.0 usb dongle plus coordinator with latest firmware.
<img width="652" alt="image" src="https://user-images.githubusercontent.com/12990053/161127977-42278b61-fa85-4ceb-bf58-bb058586c35b.png">
Also updated firmware of relay using OTA. What's the issue it could be?

**Reply by Alex3880** [reply]
Fixed this issue with older firmware 20211217, maybe it will help somebody too.

**Comment by Gmarapet** [comment]
> Fixed this issue with older firmware 20211217

I have similar issue with modern z2m, ZBDongle-E and DLKZMK11LM.
Have you update firmware of dongle recently? Or maybe can you suggest firmware with wich DLKZMK11LM working as router?

![zbdongle](https://github.com/Koenkk/zigbee2mqtt/assets/17617764/0cd53b3a-8ea5-4f3c-a3f5-24db54b0ec8d)

**Reply by flipp5b** [reply]
+1
I have the same problem with SONOFF ZBDongle-E (older firmware version 6.10.3.0) and Aqara DLKZMK11LM.
![image](https://github.com/Koenkk/zigbee2mqtt/assets/5507566/cb1b7dd4-550a-4e31-8059-b0cc5f5136e3)

![image](https://github.com/Koenkk/zigbee2mqtt/assets/5507566/4554cb91-dad7-460f-9401-749498ac6a87)
